package.json

```diff
@@ -201,17 +201,19 @@
     "watchify": "3.11.1",
     "webdriverio": "5.18.6",
     "webpack": "4.44.1",
-    "whatwg-fetch": "3.0.0"
+    "whatwg-fetch": "3.0.0",
+    "vue-template-compiler": "^2.6.11"
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "build:bundle": "node ./bin/build-bundle.js",
     "build:clean": "rm -rf packages/*/lib packages/@uppy/*/lib packages/*/dist packages/@uppy/*/dist",
     "build:companion": "cd ./packages/@uppy/companion && npm run build",
     "build:css": "node ./bin/build-css.js",
-    "build:js": "npm-run-all build:lib build:companion build:locale-pack build:bundle",
+    "build:js": "npm-run-all build:lib build:companion build:vue build:locale-pack build:bundle",
     "build:lib": "node ./bin/build-lib.js",
     "build:locale-pack": "node ./bin/locale-packs.js build",
+    "build:vue": "cd ./packages/@uppy/vue && npm run build",
```

Comment:
rename to test? or do we actually build something here?

Reply:
Nothing is built there, it's artifact that can be removed